staging/src/k8s.io/apimachinery/pkg/runtime/serializer/json/json.go

```diff
@@ -103,6 +111,8 @@ type Serializer struct {
 	typer   runtime.ObjectTyper
 
 	identifier runtime.Identifier
+
+	streamingEncoder bool
```

Comment:
Is this better suited to be a `Streaming bool` in `type SerializerOptions` above? (curious if you ruled it out already)

Reply:
I haven't yet figured out how to best pass the configuration. I run into dependency issue that apimachinery doesn't depend on apiserver featuregates. Configuring it SerializerOptions should help.

Reply:
Done